### ISSUE TITLE
Running as cronjob: "Project --project-config=/path/to/project.yml doesn't exist."

### ISSUE THREAD

**Issue by in3rsha** [tmuxinator/tmuxinator#727]
```
$ tmuxinator version
tmuxinator 1.1.2
```

I'm trying to run tmuxinator on startup by using it in a cronjob. I have my `crontab -e` set up as follows:

```
@reboot /home/user/restart.sh > ~/results.txt
```

And the `restart.sh` script looks like this:

```
#!/bin/bash
tmuxinator start --project-config=/path/to/project.yml
```

However, when I reboot my computer the cronjob results show the following error: `Project --project-config=/path/to/project.yml doesn't exist.`

The command `tmuxinator start --project-config=/path/to/project.yml` works fine when I run it myself, but it doesn't seem to work when being run as a cronjob.

Can anyone think of why this might be happening, or if it's generally possible to run `tmuxinator` on startup via a cronjob?

**Comment by ethagnawl** [comment]
Are you using a relative path to the config file? (e.g. `~/config/tmuxinator/project.yml`) If the cron job is being run as root, that won't work.

**Comment by in3rsha** [comment]
I'm using an absolute path and the cron job is for my user (`crontab -e`).

**Comment by ethagnawl** [comment]
Have you been able to verify that the file exists and is accessible from within that script? (e.g. `test -e /path/to/project.yml && echo found\ config`)

**Comment by in3rsha** [comment]
I found the problem.

I had two versions of `tmuxinator` installed. One was installed via `apt install` and the other was installed via `gem install`. I was referencing an old version of tmuxinator in my cronjob (`/usr/bin/tmuxinator`), and I'm guessing it didn't allow you to use the `--project-config` option, and was therefore interpreting this option as a project name.

I fixed the problem by uninstalling the older version `sudo apt remove tmuxinator` and explicitly invoking the new version from within the script (`/usr/local/bin/tmuxinator`).

Thank you @ethagnawl for helping me to troubleshoot. Very much appreciated!
